### ISSUE TITLE
Anvil Duplicate Enchantment Bug

### ISSUE THREAD

**Issue by KeirLoire** [badbones69/Crazy-Enchantments#362]
How to reproduce this bug:
1.) Put the two swords with enchantments that you want to combine in the Anvil.
2.) Do not get the resulting item from the Anvil GUI.
3.) Apply the Black Scroll on the resulting item from the Anvil GUI.
4.) The enchantment of the resulting item stays while the black scroll produces a copy of that enchantment.

**Comment by KeirLoire** [comment]
I made a skript to prevent this abuse:

Requires: Skript 2.2dev32c or higher
```
on rightclick on a anvil:
    loop items in player's inventory:
        if name of loop-item is "&8&l&nBlack Scroll":
            if lore of loop-item contains "&7Have a custom enchantment":
                message "&cRemove scrolls from your inventory"
                cancel event
on rightclick on a slightly damaged anvil:
    loop items in player's inventory:
        if name of loop-item is "&8&l&nBlack Scroll":
            if lore of loop-item contains "&7Have a custom enchantment":
                message "&cRemove scrolls from your inventory"
                cancel event
on rightclick on a very damaged anvil:
    loop items in player's inventory:
        if name of loop-item is "&8&l&nBlack Scroll":
            if lore of loop-item contains "&7Have a custom enchantment":
                message "&cRemove scrolls from your inventory"
                cancel event
```

**Comment by HyperVx2** [comment]
> I made a skript to counter this issue temporarily:
> 
> Requires: Skript 2.2dev32c or higher
> 
> ```
> on rightclick on a anvil:
>     loop items in player's inventory:
>         if name of loop-item is "&8&l&nBlack Scroll":
>             if lore of loop-item contains "&7Have a custom enchantment":
>                 message "&cRemove Black Scroll from your inventory."
>                 cancel event
> on rightclick on a slightly damaged anvil:
>     loop items in player's inventory:
>         if name of loop-item is "&8&l&nBlack Scroll":
>             if lore of loop-item contains "&7Have a custom enchantment":
>                 message "&cRemove Black Scroll from your inventory."
>                 cancel event
> on rightclick on a very damaged anvil:
>     loop items in player's inventory:
>         if name of loop-item is "&8&l&nBlack Scroll":
>             if lore of loop-item contains "&7Have a custom enchantment":
>                 message "&cRemove Black Scroll from your inventory."
>                 cancel event
> ```

I tried this script you made on a local server using version 2.2-dev36. It didn't seem to work.

**Comment by KeirLoire** [comment]
I don't know why it doesn't work for you but I forgot to mention that I am using the following Skript addons:
- Latest version of skQuery
- Latest version of TuSke
- Latest version of SharpSK
- Latest version of WolvSK
- Latest version of skUtils

I don't know which addon allowed the skript code to work. Try installing skQuery first.

**Comment by KeirLoire** [comment]
The code just prevents the player from using Anvil while the user has Black Scroll in their inventory.

**Comment by HyperVx2** [comment]
WolvSK was the missing addon. Thank you.
However I did modify your code by removing `if lore of loop-item contains "&7Have a custom enchantment":` line.
It works on 1.12.2 but not on 1.8.8. Possibly requires Skript 2.3 which it's not supported on 1.8.8, sadly.

**Comment by KeirLoire** [comment]
Cheers! Good luck on your server!

**Comment by badbones69** [comment]
Not sure how I never saw this. Thanks for the report and Ill work on fixing this asap.

**Comment by badbones69** [comment]
I have fixed this in the CE v1.8 Update but will work on trying to post an update for 1.7.10-1.12.2 servers as soon as I can.
